### PR TITLE
Fix crash when compiling extensionless binary items

### PR DESCRIPTION
**Pull request by ddfreyne**
Fix for #524.

### CODE REVIEW

File: lib/nanoc/cli/commands/create-site.rb [hunk 1]
```diff
@@ -130,7 +130,7 @@ def array_to_yaml(array)
     item.identifier.chop + '.css'
   elsif item.binary?
     # Write item with identifier /foo/ to /foo.ext
-    item.identifier.chop + '.' + item[:extension]
+    item.identifier.chop + '.' + (item[:extension] ? '.' + item[:extension] : '')
```
Comment (mpapis) [inline lib/nanoc/cli/commands/create-site.rb]: two dots?

Comment (ddfreyne) [inline lib/nanoc/cli/commands/create-site.rb]: Oh, doh. That’s clearly wrong.

(Must have been my lack of coffee.)

Comment (ddfreyne) [inline lib/nanoc/cli/commands/create-site.rb]: Fixed.

**Comment by mpapis** [comment]
looks good :+1:

**Pull request merged**